test/test/components/previews/preview_component_preview.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class PreviewComponentPreview < ActionView::Component::Preview
+  set_summary "Summary of the preview component"
```

Comment:
Is there an equivalent feature in ActionMailer::Preview? If not, I think we should leave this out, for now at least.

Reply:
No I don't think there is. I'll remove it